### DISCUSSION TITLE
installation

### DISCUSSION THREAD

**Discussion by prohorsev** [phalcon/cphalcon#15452] [Q&A] [answered]
I try to install according to the instructions https://docs.phalcon.io/4.0/en/upgrade#installation. On step - zephir build (I do php zephir.phar build) I get error: Internal extension compilation failed. Check compile-errors.log for more information. -> No targets specified and no makefile found. I use ubuntu20. Can anybody help me?

**Comment by Jeckerson** [comment, marked as answer]
If you are using Zephir `0.13.x`, you will need to use separate commands now:

```
zephir generate
zephir compile
```

Also, Zephir `0.13` supports only PHP >= 7.4 and designed for Phalcon v5.

**Reply by prohorsev** [reply]
Thank you.
